test/molecule-role/molecule/_shared/install-and-update/install-provisioning-dependencies.yml

```diff
@@ -2,7 +2,7 @@
 - name: Install provisioning dependencies (1/2)
   apt:
     name:
-      python3-pip
+      [python3-pip, jq]
```

Comment:
would we need to still install those 2 deps considering they are already part of the AMI (see receiver.yml) ?

Reply:
I think this file is used to provision dependencies in other instances (non-receiver), I got an error in one of the builds saying there was no `jq`.